packages/node/example/bootstrappedApp/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@inrupt/demo-app-authn-node",
+  "version": "1.0.0",
+  "description": "Demo client application for authentication in NodeJS",
+  "main": "src/index.js",
+  "scripts": {
+    "lint": "eslint \\\"src/**/*.js\\\"",
+    "start": "./node_modules/.bin/nodemon src/index.js",
+    "dev": "npm run start",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/inrupt/solid-client-authn.git"
+  },
+  "author": "Inrupt, Inc.",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/inrupt/solid-client-authn/issues"
+  },
+  "homepage": "https://github.com/inrupt/solid-client-authn#readme",
```

Comment:
This URL works, but it redirects to `https://github.com/inrupt/solid-client-authn-js#readme` (note the `-js` suffix). I'm not sure whats going on there exactly (i.e. why the redirect), but shouldn't we use the actual URL...?

Reply:
Right, my bad, it's just that somewhere down the road we renamed the GH repo to add the `-js`, and I must have copy-pasted this from somewhere not updated. Fixed in 81552d136cbe99d4bf1e305c04105e1568ff9571